### PR TITLE
Support for the google wrappers

### PR DESCRIPTION
**Pull request by olegbespalov**
# What?

This change allows the use of the [Google Protobuff wrappers](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/wrappers.proto) and brings the tests for that functionality.

# Why?

Initially reported in https://github.com/grafana/k6/issues/3232 and attempted to fix in https://github.com/grafana/k6/pull/3238

### CODE REVIEW

File: grpc/testdata/wrappers_testing/service.go
Comment (mstoykov) [inline grpc/testdata/wrappers_testing/service.go:48]:
```suggestion
	if s.TestDoublemplementation != nil {
```
?

File: grpc/testdata/wrappers_testing/service.go
Comment (mstoykov) [inline grpc/testdata/wrappers_testing/service.go:56]:
```suggestion
	if s.TestDoubleImplementation != nil {
```
?

Comment (olegbespalov) [reply grpc/testdata/wrappers_testing/service.go:56]: :facepalm:

File: lib/netext/grpcext/stream.go
Comment (mstoykov) [inline lib/netext/grpcext/stream.go:81]: nit: can we make this into a function taking the marshaller and reusing it in both places?

**Review by mstoykov — APPROVED** [review]

**Comment by mstoykov** [comment]
LGTM otherwise then us having what seems like hte same code in two places

File: lib/netext/grpcext/stream.go
Comment (mstoykov) [inline lib/netext/grpcext/stream.go:72]: nit: not certain what happened here, but I can't propose a change through the UI that seems right :(

**Review by mstoykov — APPROVED** [review]
LGTM :+1:

There is a small comment problem :(

**Review by codebien — COMMENTED** [review]
LGTM, waiting for the rebase + nit fixes for final approval

File: lib/netext/grpcext/stream.go
Comment (olegbespalov) [inline lib/netext/grpcext/stream.go:72]:
```suggestion
//		  double y = 2;
//		  double z = 3;
```

**Review by mstoykov — APPROVED** [review]

**Review by codebien — APPROVED** [review]

**Comment by olegbespalov** [comment]
For the record, I just rebased and squashed some commits, going to merge

**Pull request merged**
